src/amo/routes.js

```diff
@@ -30,6 +32,8 @@ import ServerError from './components/ErrorPage/ServerError';
 export default (
   <Route path="/:lang/:application" component={App}>
     <IndexRoute component={Home} />
+    <Route path="about" component={About} />
+    <Route path="review_guide" component={ReviewGuide} />
```

Comment:
It's preferable to use a hyphen instead of underscore in a url - sometimes underscores get lost in underlines and are less obvious.